docs/user/vulnerabilities/vulnerability/product_status-usage.en.md

```diff
@@ -1,27 +1,51 @@
 # Product Status - Usage
 
-* The same product ID must not be member of contradicting product status groups.
-  I.e. a product listed in the group of _affected_ products may not be listed again in the group of _not affected_ products.
-  The contradicting groups are
+The product status provides information on the status products in the `product_tree` are in.
 
-  * affected
+Generally, vulnerable (`known_affected`) and fixed (`fixes`) versions of products should be given.
 
-    * `product_status/first_affected[]`
-    * `product_status/known_affected[]`
-    * `product_status/last_affected[]`
+The products are listed by their ID in different lists of product status groups.
+The same product ID must not be member of contradicting product status groups.
+I.e. a product listed in the group of _affected_ products may not be listed again in the group of _not affected_ products.
 
-  * not affected
+The contradicting groups are
 
-    * `product_status/known_not_affected[]`
+* affected
```

Comment:
Please use a consistent formatting for the group names. Sometime it is with `_`, in other places without. I prefer to have a kind of emphasis as it makes it easier to realize that/when it is about the group.
Applies to the whole file / section.